pkg/importer/dataStream_test.go

```diff
@@ -267,6 +280,13 @@ func Test_dataStream_Close(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
+			buf := make([]byte, 32*1024)
```

Comment:
where did this size come from? Why is an unbound slice not used here?

Reply:
Its an arbitrary buffer size, I just want it to read until the source is exhausted (This is needed so the httpserver doesn't block waiting for a read)